pkg/util/scrubber/default.go

```diff
@@ -316,6 +316,16 @@ func ScrubLine(url string) string {
 	return DefaultScrubber.ScrubLine(url)
 }
 
+// HideKeyExceptLastFiveChars replaces all characters in the key with "*", except
+// for the last 5 characters. If the key is less than 5 characters long, replace
+// all of it with "*" instead.
+func HideKeyExceptLastFiveChars(key string) string {
```

Comment:
This is neat, but I think we should embed the 40 and 32 lenght check in the function and use `defaultReplacement` instead. The method being public we don't want to leak the actual secret size when scrubbing.

Reply:
Makes sense, done.